src/pages/docs/api/connections.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Connections
+description: Manage connections
+---
+
+Manage connections for your products and sections.
```

Comment:
Not the most self explain endpoint. The goal is to create a connection to have the provided URL call when a user try to export a bug.